projects/hslayers/src/components/data/add-data.directive.html

```diff
@@ -0,0 +1,29 @@
+<div class="card hs-main-panel">
```

Comment:
Maybe rename the file to add-data.component.html

Reply:
sure, will do